packages/expo-file-system/android/src/main/java/expo/modules/filesystem/FilePermissionModule.java

```diff
@@ -1,6 +1,7 @@
 package expo.modules.filesystem;
 
 import android.content.Context;
+import android.net.Uri;
```

Comment:
Are you sure this needs to be added to this file? This is the only change in this file 😅